ui-modules/blueprint-composer/package.json

```diff
@@ -60,7 +60,8 @@
     "lodash": "^4.16.4",
     "ngclipboard": "^1.1.1",
     "normalizr": "^2.2.1",
-    "underscore": "^1.8.3"
+    "underscore": "^1.8.3",
+    "moment": "^2.15.1"
```

Comment:
This will increase the bundle size by about 350k. There are smaller alternatives available if you just want to display the time from now like https://date-fns.org/

Reply:
nice, will try this

Reply:
works a treat